django/contrib/admin/templatetags/admin_list.py

```diff
@@ -259,8 +259,7 @@ def link_in_col(is_first, field_name, cl):
                     '<a href="{}"{}>{}</a>',
                     url,
                     format_html(
-                        ' onclick="opener.dismissRelatedLookupPopup(window, '
-                        '&#39;{}&#39;); return false;"', result_id
+                        ' data-popup-opener="{}"', result_id
```

Comment:
`result_id` should not need to go through escapejs (see above), just html escaping (which `format_html` applies) should be enough, no?